pkg/app/app_test.go

```diff
@@ -35,3 +37,44 @@ func Test_CreateAppEngine_config_replaceV1inApi(t *testing.T) {
 	actualApiUrl := config.GetString(configuration.API_URL)
 	assert.Equal(t, expectApiUrl, actualApiUrl)
 }
+
+func Test_initConfiguration_updateDefaultOrgId(t *testing.T) {
```

Comment:
I think we are missing one case here, which is where the given org_id is in a uuid pattern. This means that 'isSlugName' is false.

Reply:
added the test